plugin/templates.vim

```diff
@@ -354,6 +356,8 @@ function <SID>TExpandVars()
 	call <SID>TExpand("USER",  l:user)
 	call <SID>TExpand("FDATE", l:fdate)
 	call <SID>TExpand("MONTH", l:month)
+	call <SID>TExpand("MONTHSHORT",   l:monshort)
+	call <SID>TExpand("MONTHFULL",   l:monfull)
```

Comment:
Please remove the additional whitespace after the commas, so it looks like the lines below for e.g. `MACROCLASS` and `CAMELCLASS`.